FWCore/Framework/src/Schedule.cc

```diff
@@ -1052,7 +1065,7 @@ namespace edm {
   }
                           
   void
-  Schedule::getTriggerTimingReport(TriggerTimingReport& rep) const {
+  Schedule::getTriggerTimingReport(TriggerTimingReport& rep) {
```

Comment:
@Dr15Jones getTriggerTimingReport actually fills in the timing report, so summaryTimeKeeper is not const. This was revealed by propagate_const. I will investigate how much work it would be to make summaryTimeKeeper const, but it would be outside the scope of implementing propagate_const.